### PR TITLE
Revise the ops documents

### PR DESCRIPTION
**Pull request by loladzeus**
Revise the ops documents

### CODE REVIEW

File: docs/wip-aws-copilot.md.md
Comment (tvararu) [inline docs/wip-aws-copilot.md.md:1]: Should the file extension be `.md.md`?

File: docs/aws-copilot.md [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+AWS Copilot simplifies deploying containerized applications on AWS, automating numerous steps in the deployment process.
+
+**MacOS/Linux/WSL Installation:**
+
+1. Open your command-line interface.
+2. Use the command below:
+
+```bash
+asdf plugin add aws-copilot
+asdf install aws-copilot 1.32.0
```
Comment (misaka) [inline docs/aws-copilot.md:40]: The version is in the `.tool-versions` file, so I don't think it needs to be specified here.

```suggestion
asdf install aws-copilot
```

File: docs/aws-copilot.md [hunk 2]
```diff
@@ -1,14 +1,50 @@
-# AWS Copilot Ops Manual
+# WIP - AWS Copilot Ops Manual
```
Comment (tvararu) [inline docs/aws-copilot.md:1]:
```suggestion
# AWS Copilot Ops Manual
```

File: docs/aws-copilot.md [hunk 3]
```diff
@@ -46,72 +82,64 @@ sso_registration_scopes = sso:account:access
...
-copilot svc deploy --env staging
-```
+**If you need to tear it down**
 
-## Opening a shell on the remote environment
+`copilot app delete`, and follow the instructions
 
-If you have the service up and running, you can connect to the first running
-container with this command:
+**Setting up a new Stack**
```
Comment (tvararu) [inline docs/aws-copilot.md:97]: Don't use `**Thing**` to denote a heading, use actual markdown headings and respect the hierarchy (level 1 heading at the top, and level 2/3/4 depending on section/subsection/etc)

File: docs/wip-aws-copilot.md.md
Comment (loladzeus) [reply docs/wip-aws-copilot.md.md:1]: this has been addressed

File: docs/aws-copilot.md [hunk 4]
```diff
@@ -46,72 +82,64 @@ sso_registration_scopes = sso:account:access
...
+3. Initialise the environment:
+   `$ copilot env init --name env_name`
+4. Deploy the enviornment:
+   `copilot env deploy --name staging`
+5. initialise the secrets if you need them:
+   ` copilot secret init --name secret_name`
+6. Initialise the service:
+   `copilot svc init` and follow the instructions. In our case it is a Load Balanced Web Service option
+7. Deploy the service to the target enviornment:
+   `copilot svc deploy --name service_name --env env_name`
```
Comment (tvararu) [inline docs/aws-copilot.md:120]: This formatting doesn't look great, the commands are nested inside the bullets:

Screenshot 2024-02-12 at 13 28 32

I'd rather we do something like the cheat sheet where it's one bash section with comments for each line:

```bash
copilot app init # Initialise the app
copilot env init --name env_name # Initialise the environment
```

This makes it easier to copy and paste individual commands.

**Review by tvararu — COMMENTED** [review]
I can review again a bit later once some of the feedback is addressed 👍

**Comment by tvararu** [comment]
We'll revisit the docs later, but we can close this for now as we're mostly focusing on the product stuff instead of the docs 👍

**Pull request closed without merging**